helm_deploy/prisoner-content-hub-backend/values.yaml

```diff
@@ -6,7 +6,7 @@ maxReplicaCount: 6
 
 image:
   repository: mojdigitalstudio/prisoner-content-hub-backend
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
This is really handy for debugging & local dev, but we'll probably want to remove it in favour of specifying the version explicitly for CI (which makes rollbacks nice and fast because the image is already present). 

I wonder if there's a way to override the `pullPolicy` when deploying locally to the `development` namespace (which I'm guessing is the main/only time we'll be using the `latest` tag?

Reply:
We can add that to the environment yamls in Helm? we can revert the change in this PR and raise another for that specific behaviour?

Reply:
PR raised for this -> https://github.com/ministryofjustice/prisoner-content-hub-frontend/pull/48

Reply:
Raised a couple of draft PR's to support this behaviour
- https://github.com/ministryofjustice/prisoner-content-hub-frontend/pull/48
- https://github.com/ministryofjustice/prisoner-content-hub-backend/pull/70

Reply:
Reverted the default pull policy